_posts/2024-12-24-openttd-15-0-beta1.md

```diff
@@ -21,5 +21,5 @@ To be a bit less vague, here are some of the highlights:
 There's a lot of bugs left for you to find, though, guaranteed. Enjoy the beta, test all the things and [report anything you find amiss](https://github.com/OpenTTD/OpenTTD/issues/new/choose), to make OpenTTD 15 the best OpenTTD 15 ever.
 
 * [Download](https://www.openttd.org/downloads/openttd-releases/testing.html)
-* [Changelog](https://cdn.openttd.org/openttd-releases/15.0-beta1/changelog.txt)
+* [Changelog](https://cdn.openttd.org/openttd-releases/15.0-beta1/changelog.md)
```

Comment:
I split this off into https://github.com/OpenTTD/website/pull/334 , as fixing the empty `changelog.md` was easy, making this change a trivial fix :) The other parts of this PR are a bit more tricky :)